beacon-chain/blockchain/block_processing.go

```diff
@@ -98,6 +99,20 @@ func (c *ChainService) ReceiveBlock(ctx context.Context, block *pb.BeaconBlock)
 		log.Error("Sent processed block to no subscribers")
 	}
 
+	// Update attestation store with latest attestation target.
+	log.Info("Updating latest attestation target")
+	for _, att := range block.Body.Attestations {
+		if err := c.attsService.UpdateLatestAttestation(c.ctx, att); err != nil {
+			return nil, fmt.Errorf("failed to update latest attestation for store: %v", err)
+		}
+		log.WithFields(
+			logrus.Fields{
+				"attestationSlot":  att.Data.Slot,
+				"attestationIndex": att.AggregationBitfield,
```

Comment:
```suggestion
				"attestationBitfield": att.AggregationBitfield,
```
Shouldn't it be the same?